Zadanie 15.1 Ćwiczymy ES6/scripts.js

```diff
@@ -0,0 +1,50 @@
+// TASK 1
+
+const firstWord = 'Hello';
+const secondWord = 'World';
+
+const helloWorld = `${firstWord} ${secondWord}`;
+
+console.log(helloWorld);
+
+//or
+
+const name = (firstName = 'Maciej', secondName = 'Kubas') => console.log(`${firstName} ${secondName}`);
+name();
+
+//TASK 2
+
+const multiply = (number1 = 1, number2 = 1) => number1 * number2;
+
+
+console.log(multiply(2 * 5));
```

Comment:
chodzilo mi o to, zebys przecwiczyl laczenie tekstu i wywolania funkcji w jednym, czyli zeby efekt byl taki:
`2 * 5 = 10` (2 * 5 = to tekst na sztywno, a 10 to wynik dzialania funkcji)

Reply:
teraz ok? :)

Reply:
jeszcze nie :) bez dodatkowej zmiennej, niech to bedzie jeden console.log a w nim wszystko w jednym template stringu